rpc-darpc/src/main/java/com/ibm/crail/namenode/rpc/darpc/DaRPCConstants.java

```diff
@@ -55,6 +55,21 @@
 	public static final String NAMENODE_DARPC_CLUSTERSIZE_KEY = "crail.namenode.darpc.clustersize";
 	public static int NAMENODE_DARPC_CLUSTERSIZE = 128;	
 	
+	public static final String NAMENODE_DARPC_MEMPOOL_HUGEPAGEPATH_KEY = "crail.namenode.darpc.mempool.hugepagepath";
```

Comment:
I was under the impression we don't want this multilevel config anymore e.g. everything should be darpc.X